conan/test_package/hello_service/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(userver-samples-hello_service CXX)
```

Comment:
We are trying to imitate a user's service here, not `userver/samples`. Additionally, CMake targets may collide. Use something like `userver-conan-test-hello-service`.